### PR TITLE
docs: add release notes for v0.7.2

### PR DESCRIPTION
**Pull request by wass3r**

### CODE REVIEW

**Review by matt-fevold — COMMENTED** [review]
Should we link to the [migration doc](https://github.com/go-vela/community/tree/master/migrations/v0.7.0) in the release notes?

**Comment by JordanSussman** [comment]
Should probably add a note about being able to pass `map[string]string` data to deployments.

CLI: https://github.com/go-vela/cli/pull/171
Server: https://github.com/go-vela/server/pull/217
Types: https://github.com/go-vela/types/pull/117

**Comment by wass3r** [comment]
@JordanSussman good callout - thanks, i will add that

**Comment by wass3r** [comment]
@matt-fevold on second thought, since secret encryption didn't make it in, it might be confusing to link to that. however, there are some `ALTER` statements that need to be applied that should be called out.

**Comment by matt-fevold** [comment]
Good point @wass3r

**Comment by wass3r** [comment]
@JordanSussman added, but i don't feel like I did it justice. feel free to propose improvements. it looks like we are also missing docs on this?

File: migrations/v0.7/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# v0.7 migration
+
+_Please note: this notice applies when upgrading to the latest v0.7.x release._
+
+When migrating from Vela version [v0.6.0](https://github.com/go-vela/community/blob/master/releases/v0.6.0.md) to [v0.7.2](https://github.com/go-vela/community/blob/master/releases/v0.7.2.md) the Vela administrator will want to ensure the following actions are being performed:
+
+1. Update tables in the Postgres database
+   * `ALTER TABLE users ADD COLUMN IF NOT EXISTS refresh_token VARCHAR(500);`
+   * `ALTER TABLE builds ADD COLUMN IF NOT EXISTS deploy_payload VARCHAR(2000);`
```
Comment (JordanSussman) [inline migrations/v0.7/README.md:9]: Need to also add `ALTER TABLE workers ADD COLUMN IF NOT EXISTS build_limit INTEGER;`.

Source: https://github.com/go-vela/server/pull/219

**Comment by JordanSussman** [comment]
@wass3r - Created https://github.com/go-vela/community/issues/210 to track the documentation for that change.

File: migrations/v0.7/README.md (hunk 1 above)
Comment (wass3r) [reply migrations/v0.7/README.md:9]: added. thanks!

**Comment by wass3r** [comment]
@JordanSussman thank you! ❤️

File: releases/v0.7.2.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* Add time-sensitive auth with expiry. Eight-hour session length refreshed at 15min intervals
+* Support for writing templates in [Starlark](https://github.com/bazelbuild/starlark) (dialect of python) language
+* New Pipeline management endpoints:
+    * Expand templates within the CLI/UI
+    * Test compiling without triggering a pipeline
+* Execute a vela pipeline locally i.e. vela exec
+* Ability to modify clone process within a pipeline
+* View services natively within the UI
+
+### Enhancements:
```
Comment (JordanSussman) [inline releases/v0.7.2.md:31]: Thoughts on adding some basic screenshots to showcase the UI changes?

Comment (wass3r) [reply releases/v0.7.2.md:31]: hmm, i think a lot of the PRs had screenshots (if not all of them) - to avoid duplication maybe i could link to those.

Comment (wass3r) [reply releases/v0.7.2.md]: hmm, some of them are ok, but nothing consistent. something we can get better at. might reserve for future releases.

**Review by matt-fevold — APPROVED** [review]

**Review by jbrockopp — APPROVED** [review]
LGTM

**Pull request merged**